plugins/html/icon_html/actions/docx/action.py

```diff
@@ -16,16 +16,15 @@ def __init__(self):
     def run(self, params={}):
         temp_file = "temp_html_2_docx.docx"
         tag_parser = "(?i)<\/?\w+((\s+\w+(\s*=\s*(?:\".*?\"|'.*?'|[^'\">\s]+))?)+\s*|\s*)\/?>"  # noqa: W605
-        tags = re.findall(tag_parser, params.get("doc"))
-        try:
-            if not len(tags):
-                raise PluginException(cause="Run: Invalid input.", assistance="Input must be of type HTML.")
-            pypandoc.convert(params.get("doc"), "docx", outputfile=temp_file, format="html")
-            with open(temp_file, "rb") as output:
-                # Reading the output and sending it in base64
-                return {"docx": base64.b64encode(output.read()).decode("utf-8")}
-        except Exception:
-            return {"error": "Error occurred please try again"}
+        tags = re.findall(tag_parser, params.get(Input.DOC))
+
```

Comment:
add input as a variable here
```suggestion
docx = params.get(Input.DOC)
```
then add in the variable name rather than the `params.get` part throughout this file 😃